bynary/apps/demo/src/app/core/feature/recipe/pages/detail/recipe-detail.component.spec.ts

```diff
@@ -61,87 +76,107 @@ describe('RecipeDetailComponent', () => {
 
     describe('logic', () => {
 
-
-        it('should call getRecipe method', () => {
-            const routeParams = jest.spyOn(route.params, 'subscribe');
-
-            route.params.subscribe(() => {
-                recipesFacadeStub.getRecipe();
-                expect(recipesFacadeStub.getRecipe).toHaveBeenCalledTimes(1);
-            });
-
-            expect(routeParams).toHaveBeenCalledTimes(1);
+        it('should call getRecipe method when ngOnInit is called', () => {
+            expect(mockedRecipeFace.getRecipe).toHaveBeenCalledWith(1);
         });
 
         it('should call addIngredientsToShoppingList in onAddToShoppingList method', () => {
-            recipesFacadeStub.addIngredientsToShoppingList();
-            expect(recipesFacadeStub.addIngredientsToShoppingList).toHaveBeenCalledTimes(1);
+            component.onAddToShoppingList();
+            expect(mockedRecipeFace.addIngredientsToShoppingList).toHaveBeenCalledWith([
```

Comment:
Aufruf und Test von einander trennen für bessere Lesbarkeit
```suggestion
            component.onAddToShoppingList();
            
            expect(mockedRecipeFace.addIngredientsToShoppingList).toHaveBeenCalledWith([
```

Reply:
Meinst du hier trennen in zwei extra it-Blöcke oder nur durch eine Leerzeile?

Reply:
Sorry, meinte nur durch eine Leerzeile:)